taxi.proto

```diff
@@ -31,4 +32,4 @@ message Topup {
   uint64 asset_amount = 4; // the asset denominated amount expressed in satoshis to be used as payout. It includes also the spread as taxi service fee
   float asset_price = 5; // the price of bitcoin expressed in asset
   int32 basis_point = 6; // the spread expressed in basis point on top the amount needed to repay for bitcoin fees
```

Comment:
We could remove `basis_point` (already removed in admin.proto). Same for `millisat_per _byte` and `fee_amount` in `TopupWithAssetRequest` message since we do not use it.

Reply:
why we should remove? We still charge "service fee" on top of LBTC we provide.

ie. 1000 satoshis = 40000 USDT we need to add ie. 10% on top of it, would be better to express that to the customer.

same thinf for the fee amount.

Reply:
Re. fee_amount and millisat_per _byte

We use a fixed amount for fee that's true, but we could easily add a wider range of "fixed" fragments. ie. 250, 500, 1000, 2000 and so on.

These fields shoudl be stay in there so the user could communicate us in advance how big the current transaction is and we could pick the more similar fragment for his use case.